src/renderer.js

```diff
@@ -484,6 +484,16 @@ EPUBJS.Renderer.prototype.firstElementisTextNode = function(node) {
 	return false;
 };
 
+EPUBJS.Renderer.prototype.isGoodNode = function(node) {
+	"use strict";
```

Comment:
I think we should avoid putting it in individual functions as it is already added here (https://github.com/futurepress/epub.js/blob/master/src/base.js#L1) and the files are concatenated when built.